packages/connector-aliyun-sms/README.md

```diff
@@ -1,2 +1,31 @@
-### Aliyun SMS README
-placeholder
+# Aliyun Short Message Service
+
+Aliyun is a primary cloud service provider in Asia, offering many cloud services, including short message service (SMS for short). Aliyun Connector is a plugin provided by the Logto team to call the Aliyun SMS push service, with the help of which Logto end-users can register and sign in to their Logto account via SMS verification code.
```

Comment:
nit: 

- align with `passcode` or show both? (not sure)

    So do the content below:

    > - Remember to select "verification code" as "Applicable scene" when filling out the signature application and also "**verification code**" for "Template type" when applying for a template review because we are using these signatures and templates to send **passcode**. ……

- `including SMS (Short Message Service)` is enough? (like Wikipedia)

    <img width="435" alt="image" src="https://user-images.githubusercontent.com/10594507/175852196-f9e52fc9-5e58-42a6-9e75-d6b4957a08c2.png">

```suggestion
Aliyun is a primary cloud service provider in Asia, offering many cloud services, including SMS (Short Message Service). Aliyun Connector is a plugin provided by the Logto team to call the Aliyun SMS push service, with the help of which Logto end-users can register and sign in to their Logto account via SMS verification code (a.k.a. passcode).
```

Reply:
fixed.